package.json

```diff
@@ -50,7 +50,7 @@
     "accessible-autocomplete": "^2.0.3",
     "applicationinsights": "^1.8.8",
     "awilix": "^8.0.0",
-    "axios": "^1.2.1",
+    "axios": "^0.21.4",
```

Comment:
I think axios needs putting back to the ^1.2.1 version :)

Reply:
Fixed